app/src/main/kotlin/at/bitfire/davdroid/push/UnifiedPushReceiver.kt

```diff
@@ -76,8 +83,28 @@ class UnifiedPushReceiver: MessagingReceiver() {
                 // Later: only sync affected collection and authorities
                 collectionRepository.getSyncableByTopic(topic)?.let { collection ->
                     serviceRepository.get(collection.serviceId)?.let { service ->
+                        val syncDataTypes = mutableSetOf<SyncDataType>()
+                        if (collection.type == TYPE_ADDRESSBOOK) {
+                            // If the type is an address book, sync contacts
+                            syncDataTypes.add(SyncDataType.CONTACTS)
+                        } else {
+                            // If not an address book, must be a calendar, add events
+                            syncDataTypes.add(SyncDataType.EVENTS)
```

Comment:
If we check `supportsVTODO` below (good idea), we could check `supportsVEVENT` for events.